docs/tutorial/quick-start.md

```diff
@@ -1,201 +1,460 @@
+
+## Create your application
+
+### Scaffold the project
+
+Electron apps follow the same general structure as other Node.js projects.
+Start by creating a folder and initializing an npm package.
 
-```plaintext
-my-electron-app/
-├── package.json
-├── main.js
-├── preload.js
-└── index.html
+```sh npm2yarn
+mkdir my-electron-app && cd my-electron-app
+npm init
 ```
 
-Let's create a basic application based on the structure above.
+The interactive `init` command will prompt you to set some fields in your config.
+There are a few rules to follow for the purposes of this tutorial:
 
-#### Install Electron
+* `entry point` should be `main.js`.
+* `author` and `description` can be any value, but are necessary for
+[app packaging](#package-and-distribute-your-application).
 
-Create a folder for your project and install Electron there:
+Your `package.json` file should look something like this:
 
-```sh
-mkdir my-electron-app && cd my-electron-app
-npm init -y
-npm i --save-dev electron
+```json
+{
+  "name": "my-electron-app",
+  "version": "1.0.0",
+  "description": "Hello World!",
+  "main": "main.js",
+  "author": "Jane Doe",
+  "license": "MIT"
+}
+```
+
+Then, install the `electron` package into your app's `devDependencies`.
+
+```sh npm2yarn
+$ npm install --save-dev electron
+```
+
+> Note: If you're encountering any issues with installing Electron, please
+> refer to the [Advanced Installation][advanced-installation] guide.
+
+Finally, you want to be able to execute Electron. In the [`scripts`][package-scripts]
+field of your `package.json` config, add a `start` command like so:
+
+```json
+{
+  "scripts": {
+    "start": "electron ."
+  }
+}
+```
+
+This `start` command will let you open your app in development mode.
+
+```sh npm2yarn
+npm start
+```
+
+> Note: This script tells Electron to run on your project's root folder. At this stage,
+> your app will immediately throw an error telling you that it cannot find an app to run.
+
+[advanced-installation]: ./installation.md
+[package-scripts]: https://docs.npmjs.com/cli/v7/using-npm/scripts
+
+### Run the main process
+
+The entry point of any Electron application is its `main` script. This script controls the
+**main process**, which runs in a full Node.js environment and is responsible for
+controlling your app's lifecycle, displaying native interfaces, performing privileged
+operations, and managing renderer processes (more on that later).
+
+During execution, Electron will look for this script in the [`main`][package-json-main]
+field of the app's `package.json` config, which you should have configured during the
+[app scaffolding](#scaffold-the-project) step.
+
+To initialize the `main` script, create an empty file named `main.js` in the root folder
+of your project.
+
+> Note: If you run the `start` script again at this point, your app will no longer throw
+> any errors! However, it won't do anything yet because we haven't added any code into
+> `main.js` yet.
```

Comment:
I do this all the time, but you should pick the `yet` you prefer and remove the other one.